networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-aws.adoc

```diff
@@ -0,0 +1,53 @@
+[id="nw-control-dns-records-public-hosted-zone-aws_{context}"]
+= Controlling DNS records on an public hosted zone for AWS by using Red Hat External DNS Operator
+
+.Procedure
+
+. Export the kubeconfig file.
++
+[source,terminal]
+----
+$ export KUBECONFIG=~/<path to>/kubeconfig
+----
+
+. Check the user. The user must have access to the `kube-system` namespace. If you don’t have the credentials and want to fetch from the `kube-system` namespace then use the client.
++
+[source,terminal]
+----
+$ oc whoami
+----
++
+.Example output
+[source,terminal]
+----
+system:admin
+----
+
+. Create CR ExternalDNS for service:
++
+[source,yaml]
+----
+  apiVersion: externaldns.olm.openshift.io/v1alpha1
+kind: ExternalDNS <1>
+metadata:
+  name: sample-aws <2>
+spec:
+  provider:
+    type: AWS <3>
+  zones:
+    - "xxxx" <4>
+  Source:
+    type: Service <5>
+----
+
+<1>
+<2>
+<3>
+<4>
+<5>
+
+. Check the *AWS Console > Route53* page to ensure you added the records to the hosted zone.
+
+. Run `nslookup/dig` command to check if `service-name.<project>.svc.cluster.local` can be resolved to the service ClusterIP.
+
+. Delete the service and check if the records can be removed.
```

Comment:
Check if the dns record was recorded in the Route53 console for the routes whose domain we are filter in the CR.